test/support/mock_repo.exs

```diff
@@ -0,0 +1,5 @@
+defmodule Ecto.MockRepo do
```

Comment:
I wasn't sure about adding another `test/support` module, but it's very generic and this seemed cleaner to me than defining it in the test. I'm happy to move it to the test if that seems better to others.

Reply:
Yeah, I would prefer to keep it inside the describe in the test case, as it is not used anywhere else. Even it was, having local fixtures is often better for clarity. :)